.github/scripts/install.sh

```diff
@@ -61,29 +61,13 @@ else
 fi
 
 # Install subrepos in development mode
-for dep in $(ls external-deps)
-do
-    echo "Installing $dep subrepo"
-
-    # This is necessary to pass our minimal required version of PyLSP to setuptools-scm
-    if [ "$dep" = "python-lsp-server" ]; then
-        SETUPTOOLS_SCM_PRETEND_VERSION=`python pylsp_utils.py` pip install --no-deps -q -e external-deps/$dep
-    else
-        pip install --no-deps -q -e external-deps/$dep
-    fi
-done
+python -bb -X dev -W error install_subrepos.py
```

Comment:
Install spyder before this line.
```suggestion
python -m pip install .
python -bb -X dev -W error install_subrepos.py
```